types/trace/trace.go

```diff
@@ -41,6 +41,7 @@ type Event struct {
 	StackAddresses       []uint64     `json:"stackAddresses"`
 	ContextFlags         ContextFlags `json:"contextFlags"`
 	Args                 []Argument   `json:"args"` //Arguments are ordered according their appearance in the original event
+	ParsedArgs           []Argument   `json:"parsedArgs,omitempty"`
```

Comment:
The current convention is that we pass Nil values if arguments are missing.
Shouldn't we apply it on the parsed args too?
Open for discussion here.

Reply:
I'm not sure if you're suggesting that:
1. we should drop the `omitempty`
2. Parsed arguments should go inside Args and be nil (with a different name)
3. Something else

Could you elaborate please?

Reply:
> So I liked the idea of separating the parsed arguments from the arguments, as we don't plan to support them for long.

Actually I had no intention of removing the parsed arguments, just keeping them separated and not overriding the originals.

> I suggested here to remove the `omitempty`

This is just so when users run with `-o json` but not with `parse-arguments`, they won't see the empty field each time.

Reply:
> This is just so when users run with `-o json` but not with `parse-arguments`, they won't see the empty field each time.

So we talked previously that breaking the expected structure of events with missing arguments might sabotage efforts to put the events in databases which aren't flexible.
I ask here if it won't be a problem here as well.